src/core/config/constants.js

```diff
@@ -6,4 +6,4 @@ export const mobileContainerWidth = 700;
 /**
  * shift the actions buttons in dropdown
  */
-export const tabletContainerWidth = 895;
+export const tabletContainerWidth = 800;
```

Comment:
this is to make space between info icon & hidden button around 775px cc @nsharma123

Reply:
yes, that's fine since we reduced primary texts as well. Thanks!